src/pages/homepage/WorkComponent.jsx

```diff
@@ -0,0 +1,136 @@
+import { Fragment, useRef } from 'react';
+import { useGSAP } from '@gsap/react';
+import { NavLink } from 'react-router-dom';
+import backgroundDesign from '../../assets/homepageassets/net-design.svg';
```

Comment:
Please don't import images like this it should come from one js file

Reply:
Done sir make js file in assets

Reply:
image.js